gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -18,6 +18,67 @@
     <module name="NewlineAtEndOfFile"> <!-- Java Style Guide: Line ending: LF -->
         <property name="lineSeparator" value="lf"/>
     </module>
+    <!-- Inclusive Code checks: https://wiki.palantir.tech/display/DIVERSITY/Inclusive+Language+Glossary -->
```

Comment:
Please avoid internal links from public repositories. We should **absolutely** include our inclusive language requirements in the documentation portion of this repository, any chance you would be interested in contributing that as a separate PR?

Reply:
Apologies for getting to this so late - just saw this review. Yes, happy to! I'll put that up as a separate PR.